product_secondary_unit/views/product_views.xml

```diff
@@ -1,16 +1,20 @@
 <?xml version="1.0" encoding="utf-8" ?>
-<!-- Copyright 2018 Tecnativa - Sergio Teruel
-     License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl). -->
 <odoo>
     <record id="product_template_form_view" model="ir.ui.view">
         <field name="name">Product template Secondary Unit</field>
         <field name="model">product.template</field>
         <field name="inherit_id" ref="product.product_template_form_view" />
-        <field name="groups_id" eval="[(4, ref('uom.group_uom'))]" />
```

Comment:
Why remove the groups? Is the better way to not load the view

Reply:
Because it will raise this error `Inherited view cannot have 'Groups' define on the record. Use 'groups' attributes inside the view definition`